csharp/src/Google.Protobuf/JsonFormatter.cs

```diff
@@ -778,27 +785,50 @@ static Settings()
             /// </summary>
             public TypeRegistry TypeRegistry { get; }
 
+            /// <summary>
+            /// Whether to format enums as ints. Defaults to false.
+            /// </summary>
+            public bool FormatEnumsAsIntegers { get; }
+
             // TODO: Work out how we're going to scale this to multiple settings. "WithXyz" methods?
 
             /// <summary>
             /// Creates a new <see cref="Settings"/> object with the specified formatting of default values
-            /// and an empty type registry.
+            /// and an empty type registry. This constructor is deprecated.
             /// </summary>
             /// <param name="formatDefaultValues"><c>true</c> if default values (0, empty strings etc) should be formatted; <c>false</c> otherwise.</param>
+            [System.Obsolete]
             public Settings(bool formatDefaultValues) : this(formatDefaultValues, TypeRegistry.Empty)
             {
             }
 
             /// <summary>
             /// Creates a new <see cref="Settings"/> object with the specified formatting of default values
-            /// and type registry.
+            /// and type registry. This constructor is deprecated.
             /// </summary>
             /// <param name="formatDefaultValues"><c>true</c> if default values (0, empty strings etc) should be formatted; <c>false</c> otherwise.</param>
             /// <param name="typeRegistry">The <see cref="TypeRegistry"/> to use when formatting <see cref="Any"/> messages.</param>
+            [System.Obsolete]
```

Comment:
Possibly make this constructor just chain to the other one, just as the one above chains to this?